shared/team-building/phone-search.tsx

```diff
@@ -0,0 +1,139 @@
+        serviceMap,
+        username,
+      }
+      state = 'resolved'
+    }
+  }
+  if (state === 'none' && validity && props.teamBuildingSearchResults[phoneNumber] === undefined) {
+    state = 'loading'
+  }
+  if (state === 'none' && validity && props.teamBuildingSearchResults[phoneNumber] !== undefined) {
+    state = 'notfound'
+  }
+
+  let _onContinue = () => {
+    if (user) {
+      props.onContinue(user)
+    } else {
+      // Continue in order to start a conversation with their phone number
+      props.onContinue({
+        // substr to chop off the '+' at the start so it is in the correct format for an assertion
+        id: phoneNumber.substr(1) + '@phone',
+        prettyName: phoneNumber,
+        serviceId: 'phone',
+        serviceMap: {},
+        username: phoneNumber,
+      })
+    }
+    setPhoneNumber('')
+    setPhoneInputKey(old => old + 1)
+    setValidity(false)
+  }
+
+  return (
+    <>
+      <Kb.Box2 direction="vertical" gap="tiny" style={styles.containerStyle}>
+        <PhoneInput
+          // Supply a key to force reset the PhoneInput state after a user is added
+          key={phoneInputKey}
+          autoFocus={true}
+          onChangeNumber={onChangeNumberCb}
+          result={
+            // Pass a component into PhoneInput so it is displayed inline with the number input box
+            <>
+              {state === 'resolved' && !!user && (
+                <Kb.Box2 direction="horizontal" style={styles.foundResultContainer}>
+                  <Kb.NameWithIcon
+                    size="big"
+                    onClick={_onContinue}
+                    horizontal={true}
+                    username={user.username}
+                    metaOne={user.prettyName}
+                  />
+                </Kb.Box2>
+              )}
+              {state === 'notfound' && (
+                <Kb.Box2 direction="horizontal" style={styles.notFoundResultContainer} gap="tiny">
+                  <Kb.Box2 direction="vertical" fullHeight={true} style={styles.justifyCenter}>
+                    <Kb.Icon type="icon-placeholder-avatar-32" style={styles.placeholderIcon} />
+                  </Kb.Box2>
+                  <Kb.Box2 direction="vertical" fullHeight={true} style={styles.justifyCenter}>
+                    <Kb.Text type="BodyBig">User not found</Kb.Text>
+                  </Kb.Box2>
+                </Kb.Box2>
+              )}
+            </>
+          }
+        />
+        {state === 'loading' && <Kb.ProgressIndicator type="Small" style={styles.loading} />}
+      </Kb.Box2>
+      <Kb.Box style={styles.spaceFillingBox} />
+      <Kb.Box2 direction="horizontal" style={styles.buttonContainer} fullWidth={true}>
+        <Kb.Button style={styles.button} onClick={_onContinue} label="Continue" disabled={!validity} />
+      </Kb.Box2>
+    </>
+  )
+}
+
+const styles = Styles.styleSheetCreate(() => ({
+  button: {marginBottom: Styles.globalMargins.tiny, width: '80%'},
```

Comment:
we typically use a padding on the container to add spacing on the sides. In this case, it makes sense to wrap `PhoneSearch`'s body in something with the padding and background colors added. Then you can `width: '100%'` inside.

Reply:
Done.